autoload/arduino.vim

```diff
@@ -195,8 +200,8 @@ function! arduino#GetBuildPath() abort
     return ''
   endif
   let l:path = g:arduino_build_path
-  let l:path = substitute(l:path, '{file}', expand('%:p'), 'g')
-  let l:path = substitute(l:path, '{project_dir}', expand('%:p:h'), 'g')
+  let l:path = substitute(l:path, '{file}', substitute(expand('%:p'), '\', '/', 'g'), 'g')
+  let l:path = substitute(l:path, '{project_dir}', substitute(expand('%:p:h'), '\', '/', 'g'), 'g')
```

Comment:
Is this what you mean when you say the nested substitutes are not working? What specifically isn't working about them?

Reply:
I apologize for the confusion. Nested substitutions work fine. I will write another message about what is working as I not expecting.

I used this hack to make it work on Windows. I don't know why any path on Windows is composed using backslashes. Without changing slashes topmost `substitute` interpreted '\U' sequence as a trigger to make the next characters UPPERCASE. But on Windows path quite often starts with the `C:/User/<username>` prefix. That broke compiling and uploading.